### DISCUSSION TITLE
feature request: alert levels

### DISCUSSION THREAD

**Discussion by deltanedas** [FakeFishGames/Barotrauma#12936] [Ideas and suggestions]
since bots cant read chat, a very useful feature would be alert levels.
players with perm or on sp can change alert level to tell all bots to do things.
this would be done when there is a danger known in advance, e.g. before fighting a hunting grounds target everyone should wear a suit or if there is a very large raptor swarm everyone should have guns.

there would be 3 basic alerts:
- green, nothing special
- yellow, before you do anything ordered (manning guns fixing leaks etc) put on a diving suit. if you can't find one, complain in chat before the endworm bites off the hull and you die, so maybe someone will make a suit in time.
- red, all crew put on suits and find a weapon

technically you could do this by individually commanding each bot to prepare for expedition / find a weapon but:
- this would be much much faster since middle mouse > alert > red boom done
- less tedious with lots of bots
- bots complaining about no suit early would be useful to prevent avoidable deaths

**Comment by Hazmat616** [comment]
Yellow should be dive masks and red suits

**Comment by Hazmat616** [comment]
Perhaps yellow alert should prep the crew for external threats and red internal.
